documentation/modules/ref-prometheus-metrics.adoc

```diff
@@ -25,13 +25,18 @@ metadata:
 spec:
   kafka:
     # ...
-    metrics: {}
+    metricsConfig:
+     type: jmxPrometheusExporter
+     valueFrom:
+       configMapKeyRef:
+         name: my-config-map
+         key: my-key
     # ...
   zookeeper:
     # ...
 ----
 
-The `metrics` property might contain additional configuration for the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter^].
+The `metricsConfig` property might contain a reference to the ConfigMap containing additional configuration for the link:https://github.com/prometheus/jmx_exporter[Prometheus JMX exporter^].
 
 .Example of enabling metrics with additional Prometheus JMX Exporter configuration
```

Comment:
```suggestion
.Enabling metrics with additional Prometheus JMX Exporter configuration
```

Reply:
Maybe we have one example called _Enabling metrics using metricsConfig_ And then we add a sentence after saying something like: _If no further configuration is needed, the ConfigMap reference is empty. If further configuration is needed, the ConfigMap contains additional JMX exporter configuration._